.github/workflows/pr_validation.yml

```diff
@@ -51,10 +51,8 @@ jobs:
             .nuke/temp
             ~/.nuget/packages
           key: ${{ runner.os }}-${{ hashFiles('**/global.json', '**/*.csproj') }}
-      - name: Run './build.cmd All'
-        run: ./build.cmd All
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run './build.cmd RunTests'
```

Comment:
So a comment here - on each PR we should run:

- Build
- Tests
- NuGet packaging

The `All` target shouldn't attempt to do a NuGet __RELEASE__ without the credentials being passed in.

We'll need to do another PR to fix this.

Reply:
Got it, i'll do a fix